src/exercises/exercise-3-triangle.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Function that takes the area of a Triangle
+ * @param triangleBase - The width of the bottom of the Triangle
+ * @param triangleHeight - The height of the Triangle
+ * @returns The calculated Area of the triangle
+ */
+export const triangleArea = (
+  triangleBase: number,
+  triangleHeight: number
+): number => {
+  return 0.5 * triangleBase * triangleHeight
```

Comment:
I fear the formula can fail if you don't wrap your (triangleBase and triangleHeight) params